### DISCUSSION TITLE
size of cover image from spotify/soundcloud in thumbnail embed

### DISCUSSION THREAD

**Discussion by k4mien** [lavalink-devs/Lavalink#980] [Q&A] [answered]
as above, how can i change size of the image in embed? i know thumbnail doesnt allow to change width/heigth but somehow flavibot displays it how i want.

my output:
![Screenshot 2023-11-07 144659](https://github.com/lavalink-devs/Lavalink/assets/56881087/bb90ef5b-c2b3-4f16-888c-f5d584d4c351)

flavibot output:
![Screenshot 2023-11-07 145010](https://github.com/lavalink-devs/Lavalink/assets/56881087/e7bae092-5bf7-4ec8-a6ea-f01666567bf8)

**Comment by Nansess** [comment, marked as answer]
The size of a thumbnail is determined by the size of the image you provide. FlaviBot could be using something different to get Spotify thumbnails and the images are just smaller

**Reply by Nansess** [reply]
I just checked it out. This link is for the regular thumbnail: https://i.scdn.co/image/ab67616d0000b273227edcd51b6622cdcbe9f43c, and this one is what Flavibot displays: https://i.scdn.co/image/ab67616d00004851227edcd51b6622cdcbe9f43c. It's just getting its thumbnails from a different location

**Reply by k4mien** [reply]
ok thank you. i see the pattern of url where ..b273.. is 4851 changing it manually works. for now maybe i will using it that way
